modyn/supervisor/internal/pipeline_executor/pipeline_executor.py

```diff
@@ -823,12 +735,22 @@ def _done(self, s: ExecutionState, log: StageLog) -> None:
     @pipeline_stage(PipelineStage.POST_EVALUATION_CHECKPOINT, parent=PipelineStage.MAIN, log=False, track=False)
     def _post_pipeline_evaluation_checkpoint(self, s: ExecutionState, log: StageLog) -> None:
         """Stores evaluation relevant information so that the evaluator can be started on this pipeline run again."""
-        # we will implement this in a follow-up PR
+
+        if not s.pipeline_config.evaluation:
```

Comment:
can we ignore  s.pipeline_config.evaluation.skip_during_pipeline_execution here?

Reply:
that's an artifact from a previous version. The field `skip_during_pipeline_execution` doesn't even exist anymore.